app/controllers/books_controller.rb

```diff
@@ -2,8 +2,46 @@ class BooksController < ApplicationController
   def index
     @books = Book.all
   end
-  
+
   def show
     @book = Book.find(params[:id])
   end
+
+  def new
+    @book = Book.new
+  end
+
+  def create
+    whatever = author_params[:name].split(", ")
+    multiple_authors = []
+    whatever.each do |author|
+      new_author = Author.new(name: author)
+      if new_author.save
+        multiple_authors << new_author
+      else
+        multiple_authors << Author.find_by(name: author_params[:name])
+      end
+    end
+    book = Book.new(book_params)
+    if book.save
+      multiple_authors.each do |author|
```

Comment:
I think you could shorten this while block of code as
book.authors = multiple_authors